src/ui/wxWidgets/TreeCtrl.cpp

```diff
@@ -1012,12 +1013,23 @@ void TreeCtrl::OnRenameGroup(wxCommandEvent& WXUNUSED(evt))
   }
 }
 
+void TreeCtrl::OnStartLabelEdit( wxTreeEvent& evt )
+{
+  wxTreeItemId item = evt.GetItem();
+  if(IsReadOnly() || !item.IsOk() ||! ItemIsGroup(item)) {
```

Comment:
Why the check for !ItemIsGroup() ? We allow editing a group's name same as for an entry.

Reply:
It is a nice idea to allow editing of entry names in the list view. But for that additional handling must be added into TreeCtrl::OnStartLabelEdit() and TreeCtrl::OnEndLabelEdit(). In actual handling go TreeCtrl::OnEndLabelEdit() only renaming or adding of group is implemented (by FinishAddingGroup() or FinishRenamingGroup()). In addition in TreeCtrl::OnStartLabelEdit() the optional shown additional fields in the tree view label must be removed before editing of the name. otherwise the entry <entry> with username [xyz@abc.de] would occur as "entry [xyz@abc.de]" for editing - only entry has to be content while on place editing is active. So change the label to the entry name only in TreeCtrl::OnStartLabelEdit() and rebuild to the resulting entry, including additional shown part, in TreeCtrl::OnEndLabelEdit(). If you like to allow editing of the sub-fields as well the parser would become much more complex...

Reply:
IIRC, the Windows version does this extra parsing, but in any case this is way beyond the scope of this PR.